lib/date-utils.ts

```diff
@@ -0,0 +1,16 @@
+export const monthsString: string =
+  "Januari,Februari,Maret,April,Mei,Juni,Juli,Agustus,September,Oktober,November,Desember";
+export const monthNames: string[] = monthsString.split(",");
+
+export function getCurrentMonth(): string {
+  const monthIndex = new Date().getMonth();
+  return monthNames[monthIndex];
+}
+
+export function getCurrentYear(): number {
+  return new Date().getFullYear();
+}
+
+export function getCurrentDate(): number {
+  return new Date().getDate();
+}
```

Comment:
Can we use `Date.prototype.toLocaleString()` instead for this? It should have been supported in all major browsers nowadays, shouldn't it?

I'm also using it at [the home page](https://github.com/kawalcovid19/wargabantuwarga.com/blob/ba189537c16236765df08078b9d154aafcd27c2f/pages/index.tsx#L44).

That way, I think we can simplify the implementation of the functions in this `date-utils.ts` file.

Reply:
Done